Fix icon colours on dark mode

amycatgirl opened the pull request:
Some icons, mainly the ones present on the space sidebar and the home page, are not themed properly and are missing a `fill` attribute, which causes contrasts issues as the default colour is black. And as we all know, black on dark colours don't bode well on dark mode themes.

Commit:
chore: update submodule

Commit:
chore: update lockfile

Commit:
fix: wrong colours on dark mode (backport from previous commits)

Rexogamer commented on packages/client/components/app/interface/settings/_layout/Sidebar.tsx:
```diff
@@ -9,8 +9,8 @@ import { useSettingsNavigation } from "../Settings";
   SidebarButton,
-  SidebarButtonContent,
-  SidebarButtonTitle,
+  SidebarButtonContent, SidebarButtonIcon,
```
you'll probably want to run Prettier

amycatgirl replied:
I wanna kill webstorm for that

amycatgirl commented:
crap i forgot to exclude submodules on the fmt command hold on

Commit:
fmt: EVERYTHING
turns out that the fmt and fmt:check scripts were broken, now they aren't

ok i fixed the previous commit, i am dumb

amycatgirl commented:
OK submodules now shouldn't error on CI

retcinder commented:
![image](https://github.com/user-attachments/assets/3c266a26-c987-4552-9463-8d46fe92b3f8)
![image](https://github.com/user-attachments/assets/fb00830f-466e-4f77-b6f4-bbf8192b5487)

These could be changed to be lighter (along with first one maybe having a background all the time?), otherwise ACK.

Rexogamer committed:
fix: apply colours correctly to settings close button

Rexogamer commented:
[Screencast From 2025-04-01 21-12-07.webm](https://github.com/user-attachments/assets/b54140eb-8d29-4227-900f-5f7dd5db0dd0)

retcinder reviewed (approved):
ACK, my other complaint doesn't really need to be resolved (the lighter background makes dark colours for channel invite/settings icon not bad)

Rexogamer committed:
chore(git): merge main into branch
